src/base58.c

```diff
@@ -44,8 +45,9 @@ int btc_base58_decode(void* bin, size_t* binszp, const char* b58)
     size_t binsz = *binszp;
     const unsigned char* b58u = (const void*)b58;
     unsigned char* binu = bin;
-    size_t outisz = (binsz + 3) / 4;
-    uint32_t outi[outisz];
+    const size_t outisz = (binsz + 3) / 4;
+    const size_t outilen = sizeof(uint32_t) * outisz;
+    uint32_t *outi = (uint32_t *)alloca(outilen);
```

Comment:
Then I'll replace alloca() to fixed-size arrays where possible and to btc_malloc()/btc_free() where not.